tests/containers/podman_pods.pm

```diff
@@ -81,11 +81,17 @@ sub run {
         install_k3s();
         record_info('Test', 'kube apply');
         assert_script_run('podman kube apply --kubeconfig ~/.kube/config -f pod.yaml');
-        assert_script_run('kubectl wait --for=condition=Ready pod/testing-pod');
+        assert_script_run('kubectl wait --for=condition=Ready pod/testing-pod', timeout => 240);
         validate_script_output('kubectl exec testing-pod -- cat /etc/os-release', sub { m/SUSE Linux Enterprise Server/ });
+        uninstall_k3s();
```

Comment:
I would put this after `$podman->cleanup_system_host();` or even better into a cleanup routine, which is executed also on test failures.